src/main/kotlin/dev/arbjerg/ukulele/command/PlayCommand.kt

```diff
@@ -20,8 +22,14 @@ class PlayCommand(
 ) : Command("play", "p") {
     override suspend fun CommandContext.invoke() {
         if (!ensureVoiceChannel()) return
-        val identifier = argumentText
-        apm.loadItem(identifier, Loader(this, player, identifier))
+        var identifier = argumentText
```

Comment:
This can be simplified a lot. Using exception handling as control flow is an antipattern.
```kotlin
val identifier = if(argumentText.startsWith("http")) argumentText else "ytsearch:$identifier"
```

Reply:
A more robust solution would be `argumentText.startsWith("http://") || argumentText.startsWith("https://")`

Reply:
I can't argue with that now 😄